src/libsync/vfs/cfapi/hydrationjob.h

```diff
@@ -121,6 +119,11 @@ public slots:
     int _errorCode = 0;
     int _statusCode = 0;
     QString _errorString;
+    QString _remoteParentPath;
+
+    // TODO: Create a base class with this member in 'protected' and virtual slots for fetch/upload, then refactor this and other similar classes to inherit the
+    // base class
```

Comment:
either do what is in the todo or do not tell everybody that your work is not done